examples/workflow/README.md

```diff
@@ -0,0 +1,66 @@
+# Dapr Workflow Example with go-sdk
+
+## Step
+
+### Prepare
+
+- Dapr installed
+
+### Run Workflow
+
+<!-- STEP
+name: Run Workflow
+output_match_mode: substring
+expected_stdout_lines:
+  - '== APP == Runtime initialized'
+  - '== APP == TestWorkflow registered'
+  - '== APP == TestActivity registered'
+  - '== APP == runner started'
+  - '== APP == workflow started with id: a7a4168d-3a1c-41da-8a4f-e7f6d9c718d9'
+  - '== APP == workflow paused'
+  - '== APP == workflow resumed'
+  - '== APP == stage: 1'
+  - '== APP == workflow event raised'
+  - '== APP == stage: 2'
+  - '== APP == workflow status: COMPLETED'
+  - '== APP == workflow purged'
+  - '== APP == stage: 2'
+  - '== APP == workflow started with id: a7a4168d-3a1c-41da-8a4f-e7f6d9c718d9'
+  - '== APP == workflow terminated'
+  - '== APP == workflow purged'
+background: true
+sleep: 60
+-->
+
+```bash
+dapr run --app-id workflow \
+         --dapr-grpc-port 50001 \
+         --log-level debug \
+         --resources-path ./config \
+         -- go run ./main.go
+```
+
+<!-- END_STEP -->
+
+## Result
+
+- workflow
```

Comment:
can be removed.

Reply:
Please do not remove the who result output ... only this line workflow can be removed.

Reply:
Reverted the change, thanks for catching.